fastlane_core/lib/fastlane_core/provisioning_profile.rb

```diff
@@ -62,7 +62,7 @@ def install(path)
           end
         end
 
-        true
+        destination
```

Comment:
return value is changed.
However, it seems not to be used anywhere.

https://github.com/fastlane/fastlane/blob/master/gym/lib/gym/detect_values.rb#L59
https://github.com/fastlane/fastlane/blob/master/sigh/lib/sigh/manager.rb#L41